src/components/search-textbox.vue

```diff
@@ -0,0 +1,58 @@
+<!--
+Copyright 2025 ODK Central Developers
+See the NOTICE file at the top-level directory of this distribution and at
+https://github.com/getodk/central-frontend/blob/master/NOTICE.
+
+This file is part of ODK Central. It is subject to the license terms in
+the LICENSE file found in the top-level directory of this distribution and at
+https://www.apache.org/licenses/LICENSE-2.0. No part of ODK Central,
+including this file, may be copied, modified, propagated, or distributed
+except according to the terms contained in the LICENSE file.
+-->
+
+<template>
+  <label class="form-group">
+    <input v-model="searchTextbox" class="form-control search-textbox" :placeholder="label"
+      :aria-label="label" :aria-disabled="disabled" autocomplete="off" @keydown.enter="setSearchTerm" @focusout="revert">
+    <button v-show="searchTextbox" type="button" class="close"
+      :aria-label="$t('action.clearSearch')" @click="clearSearch">
+      <span aria-hidden="true">&times;</span>
+    </button>
+    <span v-if="!noLabel" class="form-label">{{ label }}</span>
+  </label>
+</template>
+
+<script setup>
+import { ref, watch } from 'vue';
+
+
+const props = defineProps({
+  modelValue: String,
+  disabled: Boolean,
+  /**
+   * Text for label and placeholder
+   */
+  label: String,
+  noLabel: Boolean
```

Comment:
I see that the `SearchTextbox` in `EntityList` specifies props for both `label` and `noLabel`. That makes me wonder about the following:

1. Is the `label` prop always expected? I think so, because it will used in the `placeholder` and `aria-label`. If that's right, how about specifying `required: true` for the prop?
2. If the `label` prop is always expected, I think the name `noLabel` is slightly confusing. What do you think about a name like `hideLabel` or `hiddenLabel`? It seems like that prop has to do with whether a `.form-label` element is rendered, not whether there's a label at all.